src/scene/scene.js

```diff
@@ -705,7 +705,7 @@ class Scene extends EventHandler {
         this._skyboxMip = render.skyboxMip === undefined ? 0 : render.skyboxMip;
 
         if (render.skyboxRotation) {
-            this._skyboxRotation.setFromEulerAngles(render.skyboxRotation[0], render.skyboxRotation[1], render.skyboxRotation[2]);
+            this.skyboxRotation = (new Quat()).setFromEulerAngles(render.skyboxRotation[0], render.skyboxRotation[1], render.skyboxRotation[2]);
```

Comment:
Just out of curiosity, why is quat enclosed with brackets here?

Reply:
No particular reason other than I like to be explicit in this case.

I suppose `new Quat().setFromEulerAngles()` is pretty standard, but still looks a little strange to me.